src/main/resources/META-INF/maven/site-macros.vm

```diff
@@ -110,21 +110,21 @@
 #*    *##set ( $imgAlt = ' alt=""' )
 #*  *##end
 #*  *##if( $border )
-#*    *##set ( $imgBorder = ' border="' + $border + '"' )
+#*    *##set ( $imgBorder = 'border: ' + $border + ';' )
```

Comment:
This should be rather `border-width` (https://developer.mozilla.org/en-US/docs/Web/CSS/border-width) as the css attribute `border` may also set other styles and may lead to some ambiguity and `border-width` is the equivalent of the `border` html attribute. The spec is not really clear in this regard though:  https://maven.apache.org/doxia/doxia-sitetools/doxia-decoration-model/decoration.html#class_bannerLeft

Reply:
I agree this needs clarification or change, but that will happen in Doxia 2 only.

Reply:
Raise an issue with Doxia Sitetools.

Reply:
Done in https://issues.apache.org/jira/browse/DOXIASITETOOLS-253.